docs/book/how-to/setting-up-a-project-repository/stacks-pipelines-models.md

```diff
@@ -0,0 +1,85 @@
+
+- reduces the overhead of configuring your infrastructure every time you run a pipeline.
+- provides a consistent environment for your pipelines to run in, promoting reproducibility.
+- reduces risk of errors when it comes to what hardware and tool configurations to use.
+
+As for Models, they are independent of the stack used to create them. Models in ZenML help you tie resources from multiple pipelines together, as you will learn in the next section.
+
+
+## How do I organize my Pipelines, Models, and Artifacts?
+
+Pipelines, Models, and Artifacts form the core of your ML workflow in ZenML. All of your project logic is organized around these concepts and as such, it helps to understand how they interact with each other and how to structure your code to make the most out of them.
+
+### Pipelines
+
+A pipeline typically encompasses the entire ML workflow, including data preparation, model training, and evaluation. It's a good practice to have a separate pipeline for different tasks like training and inference. This makes your pipelines more modular and easier to manage.
+
+- separation of pipelines by the nature of the task allows you to run them independently as needed. For example, you might train a model in a training pipeline only once a week but run inference on new data every day.
+- it becomes easier to manage and update your code as your project grows more complex.
+- different people can work on the code for the pipelines without interfering with each other.
+- it helps you organize your runs better.
+
+### Models
+
+Models are what tie related pipelines together. A Model in ZenML is a collection of data artifacts, model artifacts, pipelines and metadata that can all be tied to a specific project.
+As such, it is good practice to use a Model to move data between pipelines.
+
+Continuing with the example of a training and an inference pipeline, you can use a ZenML Model to handover the trained model from the training pipeline to the inference pipeline. The Model Control Plane allows you to set Stages for specific model versions that can help with this.
+
+### Artifacts
+
+Artifacts are the output of a pipeline step that you want to track and reuse across multiple pipelines. They can be anything from a dataset to a trained model. It is a good practice to name your artifacts appropriately to make them easy to identify and reuse. Every pipeline run that results in a unique execution of a pipeline step produces a new version of your artifact. This ensures that there's a clear history and traceability of your data and model artifacts.
+
+Artifacts can be tied to a Model for better organization and visibility across pipelines. You can choose to log metadata about your artifacts which will then show up in the Model Control Plane.
+
+## So how do I put this all together?
+
+![Diagram showing how Models bring together Pipelines and Artifacts](../../.gitbook/assets/model_pipeline_artifact.png)
```

Comment:
This diagram doesnt really relate to the text below. might be worth spending a bit of effort making it match)

Reply:
@strickvl are you working on this one?

Reply:
i can take this

Reply:
![Model Stacks Pipelines](https://github.com/user-attachments/assets/12eb6cb2-6769-4f72-bbb6-5782d79f8024)
how about something like this? @htahir1

Reply:
it's in the Media Assets file with name "Model Stacks Pipelines"

Reply:
Can you ask @znegrin to take a crack at it? not sure about the Bob and Alice logos and zuri probably has some ideas.. @znegrin can you read the text below and give input as to how to model this?